src/user-script-obj.js

```diff
@@ -249,7 +249,11 @@ window.EditableUserScript = class EditableUserScript
     // of the generated parts.
     this._evalContent
         // Note intentional lack of line breaks before the script content.
-        = `try { (function scopeWrapper(){ function userScript() { ${this._content} }
+        = `try {
```

Comment:
Please see the comment on the line above.

Adding a line after `_content` is good to fix this bug.  Adding lines above it makes the lines in reported errors harder to use.

Reply:
Ah, missed that. I'll submit a commit the removes the lines.